presto-docs/src/main/sphinx/release/release-0.252.rst

```diff
@@ -0,0 +1,40 @@
+=============
+Release 0.252
+=============
+
+**Highlights**
+==============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix hanging join operator when spill is enabled and the probe side finishes before the hash builder starts.
+* Add file based password authenticator plugin.
+* Add new spilling strategy to spill when the user + system + revocable memory on a node exceeds max_total_memory_per_node.  This can be enabled by setting ``experimental.spiller.task-spilling-strategy`` to ``PER_QUERY_MEMORY_LIMIT``.
```

Comment:
```
Add new spilling strategy to spill when a combination of user, system and recovable memory on a node exceeds ``max_total_memory_per_node``. This can be enabled by setting the system configuration ``....`` to ``...``.
```